src/AssetsTransferApi.spec.ts

```diff
@@ -478,4 +555,38 @@ describe('AssetTransferAPI', () => {
 			);
 		});
 	});
+
+	describe('multiLocationAssetIsDestParachainsNativeAsset', () => {
+		it('Should correctly return true when a foreign assets multilocation matches its native chain of origin', () => {
+			const expected = true;
+
+			const multiLocationAssetId =
+				'{"parents":"1","interior":{"X1": {"Parachain":"2023"}}}';
+			const destChainId = '2023';
+
+			const isNativeChain =
+				systemAssetsApi.multiLocationAssetIsDestParachainsNativeAsset(
+					destChainId,
+					multiLocationAssetId
+				);
+
+			expect(isNativeChain).toEqual(expected);
```

Comment:
For this one expect(isNativeChain).toEqual(false); instead of saving this to a variable. Similar to the above.